messaging/broker.go

```diff
@@ -1204,7 +1260,7 @@ func (r *TopicReader) Read(p []byte) (int, error) {
 		if err == ErrReaderClosed {
 			return 0, io.EOF
 		} else if err != nil {
-			return 0, fmt.Errorf("file: %s", err)
```

Comment:
Yeah, this is what I am trying to do. Up for a better approach, but I do need to catch this specific error.